daily-challenges/2024-11/28/min241128.spec.ts

```diff
@@ -0,0 +1,34 @@
+/*
+# Playwright challenge
+Túi mù, hay còn gọi là “blind bag” hoặc “mystery bag,” là loại sản phẩm chứa những món đồ ngẫu nhiên, khách hàng không biết trước nội dung bên trong cho đến khi mở túi.
+
+# Đề bài:
+Hãy viết test cho đề bài sau:
+- Truy cập trang: https://material.playwrightvn.com/
+- Click vào: Game 04: Túi mù pokemon
+- Chọn đúng pokemon theo yêu cầu. Khi tìm được Pokemon, hãy dừng lại.
+*/
+
+import {test, expect} from '@playwright/test'
```

Comment:
Please format code

Reply:
sorry for the late response. code is updated.